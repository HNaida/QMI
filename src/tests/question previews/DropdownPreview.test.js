import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react'
import DropdownPreview from '../../components/previews/DropdownPreview'

configure({ adapter: new Adapter() });

describe('<DropdownPreview/>', () => {
    it('renders a dropdown question', () => {
        const question= {
            id: "v5",
            type: "dropdown",
            title: "untitled dropdown",
            options: [
                {title: "hobby/sport", numeric_value: 0},
                {title: "werk", numeric_value: 25},
                {title: "vriendschap", numeric_value: 50},
            ],
        };
        const component = shallow(<DropdownPreview question={question}/>);
        expect(component).toMatchSnapshot();
    });
});