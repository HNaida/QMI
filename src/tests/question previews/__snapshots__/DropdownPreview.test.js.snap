// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropdownPreview/> renders a number question 1`] = `ShallowWrapper {}`;
